### ISSUE TITLE
Details and Graph tabs don't work

### ISSUE THREAD

**Issue by agarwal** [OCamlPro/ows#13]
When clicking a package, a modal pops up with 3 tabs on the left. By default, we're on Overview. Nothing happens when I click the other two tabs, Details and Graph. I'm on a Mac OS X and checked on both Chrome and Firefox.

**Comment by abate** [comment]
yes. this is by design (and I'm very bad at web design) . The tabs are empty at the beginning . By clicking on the icons associated to each version, each tabs is populated with the explanations and the graph. Only the overview tab has a click-able link to get back to the overview. Confusing I know, but  I didn't find a way to to it better. Suggestions are welcome.

**Comment by agarwal** [comment]
I see clicking on the graph icons does show a graph on Firefox, but nothing shows up on Chrome. But still it is kind of confusing.  :)  Are there supposed to be any Details showing also?

**Comment by abate** [comment]
The details part should show the text explanation. But this part has still rough edges. The idea is to integrate a better explanation engine directly in dose. There is still a bit of work ahead.
